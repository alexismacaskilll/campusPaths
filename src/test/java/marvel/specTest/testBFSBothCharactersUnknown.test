# Test the Find Path with two nodes that are not in the graph

# Load a graph from tsv file that contains two heros in same book and one hero in a
# different book.
LoadGraph graph1 threeHerosTwoBooks.tsv

# Find shortest path from hero4 to hero5
FindPath graph1 hero4 hero5