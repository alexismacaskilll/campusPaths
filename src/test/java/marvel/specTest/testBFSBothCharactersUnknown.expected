# Test the Find Path with two nodes that are not in the graph

# Load a graph from tsv file that contains two heros in same book and one hero in a
# different book.
loaded graph graph1

# Find shortest path from hero4 to hero5
unknown character hero4
unknown character hero5